### ISSUE TITLE
stereo3d.geom: Couldn't create shader

### ISSUE THREAD

**Issue by nobicycle** [performous/performous#483]
$ performous -l DEBUG
see log attached
[infolog.txt](https://github.com/performous/performous/files/3569605/infolog.txt)

### Output of `performous --version`: (_What version of Performous are you using?_)
$ performous --version
Performous 1.1+

### What is your environment & configuration (arguments, platform, ...)?

$ uname -a
Linux tower 5.2.8-1-MANJARO #1 SMP PREEMPT Fri Aug 9 19:34:20 UTC 2019 x86_64 GNU/Linux
A USB webcam was plugged in. Pulseaudio applet showed the mike working fine.

> $ performous -l DEBUG
> ALSA lib pcm_dsnoop.c:638:(snd_pcm_dsnoop_open) unable to open slave
> ALSA lib pcm_dmix.c:1108:(snd_pcm_dmix_open) unable to open slave
> ALSA lib pcm.c:2564:(snd_pcm_open_noupdate) Unknown PCM cards.pcm.rear
> ALSA lib pcm.c:2564:(snd_pcm_open_noupdate) Unknown PCM cards.pcm.center_lfe
> ALSA lib pcm.c:2564:(snd_pcm_open_noupdate) Unknown PCM cards.pcm.side
> connect(2) call to /dev/shm/jack-1000/default/jack_0 failed (err=No such file or directory)
> attempt to connect to server failed
> connect(2) call to /dev/shm/jack-1000/default/jack_0 failed (err=No such file or directory)
> attempt to connect to server failed
> ALSA lib pcm_oss.c:377:(_snd_pcm_oss_open) Unknown field port
> ALSA lib pcm_oss.c:377:(_snd_pcm_oss_open) Unknown field port
> ALSA lib pcm_usb_stream.c:486:(_snd_pcm_usb_stream_open) Invalid type for card
> ALSA lib pcm_usb_stream.c:486:(_snd_pcm_usb_stream_open) Invalid type for card
> ALSA lib pcm_dmix.c:1108:(snd_pcm_dmix_open) unable to open slave
> connect(2) call to /dev/shm/jack-1000/default/jack_0 failed (err=No such file or directory)
> attempt to connect to server failed
> audio/error: Audio device 'dev="USBMIC" mics="blue,red"': No such device.
> audio/error: Audio device 'dev="Microphone" mics="*"': No such device.
> audio/error: Audio device 'mics="blue"': Device doesn't have enough input channels
> opengl/error: Invalid enum in Shader::compile glCreateShader
> FATAL ERROR: stereo3d.geom: Couldn't create shader.
> core/error: stereo3d.geom: Couldn't create shader.
> More details might be available in "/home/me/.cache/performous/infolog.txt".
> If you think this is a bug in Performous, please report it at 
>   https://github.com/performous/performous/issues
>

**Comment by Lord-Kamina** [comment]
That doesn't look like an actual debug log. Try --log debug or --loglevel debug.

Also, where did you get that performous from? Some repository? Built yourself?

<sub>Sent with <a href="http://githawk.com">GitHawk</a></sub>

**Comment by Tronic** [comment]
Possible duplicate of #480

**Comment by Baklap4** [comment]
Closing in favor of #480 

TL;DR; clone the repo, build from source:
```
# update system
sudo apt -y update
# install dependencies
sudo apt -y install cmake gettext help2man clang gcc-7 g++-7 libepoxy-dev libsdl2-dev libcairo2-dev libpango1.0-dev librsvg2-dev libboost-all-dev libavcodec-dev libavformat-dev libswscale-dev libswresample-dev libpng-dev libjpeg-dev libxml++2.6-dev portaudio19-dev libopencv-dev libportmidi-dev libqrencode-dev libicu-dev libcpprest-dev libglm-dev libopenblas-dev libfftw3-dev
#clone repo & build.
git clone git@github.com:performous/performous.git
cd performous
mkdir build
cd build
cmake ..
make -j2
sudo make install
```
